src/ui/Quickview/Quickview.ts

```diff
@@ -259,6 +254,53 @@ export class Quickview extends Component {
     }
   }
 
+  private buildContent() {
+    const icon = this.buildIcon();
+    const { caption, arrow } = this.buildCaption();
+    const content = $$('div');
+
+    content.append(icon);
+    content.append(caption);
+    $$(this.element).append(content.el);
+
+    const popperReference = this.buildPopper(icon, caption, arrow);
+    $$(this.element).on('mouseover', () => {
+      popperReference.update();
+    });
+  }
+
+  private buildIcon() {
+    const icon = $$('div', { className: 'coveo-icon-for-quickview' }, SVGIcons.icons.quickview).el;
+    SVGDom.addClassToSVGInContainer(icon, 'coveo-icon-for-quickview-svg');
+    return icon;
+  }
+
+  private buildCaption() {
```

Comment:
I would adjust the name here to communicate that it is also building an arrow. e.g.
`buildCaptionWithArrow()`? 
or
`buildCaptionAndArrow`?

Reply:
I will put this part in buildTooltip directly